### ISSUE TITLE
Odd extra "subdataset(impossible)" status record on  get -n subds

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#6208]
```shell
(git-annex)lena:~/datalad/dandi/dandisets[draft]git-annex
$> datalad get  -n 000017                                 
subdataset(ok): 000017 (dataset)
[INFO   ] Remote origin not usable by git-annex; setting annex-ignore                                                                                                                                                
[INFO   ] https://github.com/dandisets/000017.git/config download failed: Not Found                                                                                                                                  
[INFO   ] access to 1 dataset sibling dandi-dandisets-dropbox not auto-enabled, enable with:
| 		datalad siblings -d "/home/yoh/datalad/dandi/dandisets/000017" enable -s dandi-dandisets-dropbox 
install(ok): /home/yoh/datalad/dandi/dandisets/000017 (dataset) [Installed subdataset in order to get /home/yoh/datalad/dandi/dandisets/000017]
subdataset(impossible): /home/yoh/datalad/dandi/dandisets/000017 [path not contained in any matching subdataset]

$> datalad --version
datalad 0.15.2
```
seems to be benign but why it is there at all?

**Comment by adswa** [comment]
Should be fixed in an upcoming release by https://github.com/datalad/datalad/pull/6094 I guess?

**Comment by adswa** [comment]
Oh no, should be released already, should be fixed in ``0.15.3``. Please reopen if I'm mistaken

**Comment by yarikoptic** [comment]
FWIW, for that PR commit
```
$> git describe --contains 496378606eedcd54e0960b8c013a6416ad806013
0.15.3~8^2
```
so indeed should be in 0.15.3 . Thank you! 

PS for some reason conda-forge stopped automagically picking up new datalad releases on pypi and submitting PRs -- need to update in conda "manually" to get 0.15.3 release in
